diffusion/models/models.py

```diff
@@ -273,7 +351,8 @@ def stable_diffusion_xl(
         model = DeviceGPU().module_to_device(model)
         if is_xformers_installed:
             model.unet.enable_xformers_memory_efficient_attention()
-            model.vae.enable_xformers_memory_efficient_attention()
+            if autoencoder_path is None:
```

Comment:
Shouldn't this be a hasattr check instead of autoencoder_path is none check? I don't follow.

Reply:
+1 hasattr check would be better

Reply:
Agreed, it's a `hasattr` now.